bndtools.core/resources/unprocessed/serviceTemplates/java-service-consumer/bnd.bnd

```diff
@@ -0,0 +1,12 @@
+-buildpath: \
+	osgi.annotation,\
+	org.osgi.service.component.annotations,\
+	$api_package$
+
+$if(javaLevel)$
```

Comment:
hmm, I'd prefer not to see these details on project levels but let them be handled on workspace level.

Reply:
Again I don't get what you mean workspace-level...when these service projects could be created in any workspace (?)

Reply:
The reason I don't like this...with the current new project wizard and/or the new service projects wizard...is that on the first wizard page the user is asked what/which java level they want for the newly created project/projects.  I would seem to me to be shocking for the user if they were to set to something specific and then not have it set to what they specified for each project...or to have the workspace javalevel be reset by the creation of a new project (and break everything they would have there already).